examples/socketio/src/application.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-socketio
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BootMixin} from '@loopback/boot';
+import {ApplicationConfig} from '@loopback/core';
+import {RepositoryMixin} from '@loopback/repository';
+import {ServiceMixin} from '@loopback/service-proxy';
+import {SocketIoApplication, SocketIoServer} from '@loopback/socketio';
+import debugFactory from 'debug';
+import {SocketIoController} from './controllers';
+
+const debug = debugFactory('loopback:example:socketio:demo');
+
+export {ApplicationConfig};
+
+export class SocketIoExampleApplication extends BootMixin(
+  ServiceMixin(RepositoryMixin(SocketIoApplication)),
+) {
```

Comment:
We can get rid of ServiceMixin/RepositoryMixin as this example does not call backend services and databases.